### PR TITLE
Publish everything

### PR DESCRIPTION
**Pull request by gleb-lobov**
Caveats:
1. Will update things that were not updated locally - the diffing is incoming in next PR
2. Does not delete things - should we add it?
3. publishing cardinality other than 1 or 0 in source apps breaks frontend
4. publishing an email  without a top level domain is accepted by backend, but not by frontend

### CODE REVIEW

File: internal/publish/logic_dp.go [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+			return err
+		}
+	}
+	return nil
+}
+
+func PrintChangeset(changes DataProductChangeSet) {
+	if len(changes.saCreate) != 0 {
+		for _, sa := range changes.saCreate {
+			slog.Info("will create source apps", "name", sa.Name, "id", sa.Id)
```
Comment (jamessnowplow) [inline internal/publish/logic_dp.go:215]: should probably use the logging pattern we have elsewhere,`slog.Info("publish", "msg", "will xxxx", ..args)`

**Review by jamessnowplow — COMMENTED** [review]
I'm guessing you are adding a call to validate later?

While I was in here I saw one we missed on the download. Source apps have a `data.entities.tracked[_].comment` property. We are dropping it at the moment.

**Comment by jamessnowplow** [comment]
> Does not delete things - should we add it?

we should, but make it an explicit command. Some kind of `cli dp purge` or `trim` or something.

> publishing cardinality other than 1 or 0 in source apps breaks frontend

something to add to cli validation probably. I think source apps working like that was an intentional choice. 

> publishing an email without a top level domain is accepted by backend, but not by frontend

I guess we are having a validation fight and updating one of them :)

**Review by jamessnowplow — APPROVED** [review]
sorry, meant to approve

**Comment by gleb-lobov** [comment]
>Source apps have a data.entities.tracked[_].comment property. We are dropping it at the moment.

Is it exposed in the UI anywhere? I can seem to find it

**Pull request merged**